web/src/components/njwds-extended/Alert.tsx

```diff
@@ -18,12 +20,21 @@ export const AlertVariants = ["info", "success", "warning", "error", "note"] as
 
 export type AlertVariant = (typeof AlertVariants)[number];
 
+const composeAriaGroupLable = (content: string, alertMappedName: string): string => {
+  return modifyContent({
+    content,
+    condition: () => true,
+    modificationMap: {
+      alertVariant: alertMappedName,
+    },
+  });
+};
+
 export const Alert = (props: Props): ReactElement => {
   const { variant, children, noIcon, heading, rounded, dataTestid } = props;
   const variantClass = variant ? `usa-alert--${variant}` : "";
   const noIconClass = noIcon ? " usa-alert--no-icon" : "";
   const roundedClass = rounded ? " radius-md" : "";
-  const alertRole = variant === "error" ? { role: "alert" } : {};
```

Comment:
dust - removed this role as when the element is inlined I think announcing it as a group is better and the popup version has separate logic to make sure it's spoken out loud as an alert.